### ISSUE TITLE
[Bug]: Image super zoomed in cutscenes and under water

### ISSUE THREAD

**Issue by Sandritosway** [nuggslet/MGSM2Fix#23]
Hi,
i don't know if it's the same bug reported about Snake not visible when under water but in my case with the internal resolution on all cutscenes are like super zoomed and you can only see one point of the scene while they're playng and the same happens when you go into the water. When game starts, for example, if you enter in the water the image is zoomed on the wall near the stairs and the model of the character is still present but you can only see it when Snake walks in front of the camera.

(nothing changes if the screen size is 4:3, 16:9 or anything else)

**Comment by nuggslet** [comment]
Yeah, this is a duplicate of #18 but I'll keep this one as it's more descriptive

**Comment by AzadNezamdoust** [comment]
I have the problem to.

**Comment by Svila221** [comment]
Same problem here.

**Comment by Blaskoguy** [comment]
same.

**Comment by quim024** [comment]
Is there any solution to this?

**Comment by Bigswagyea** [comment]
Jus made an account to confirm this is still a thing. maybe im not doing the settings right but its as soon as i turn internal resolution on the steam deck it shows cutscenes in 2 layers. And when i first started playing and went into the water it zoomed super close.

**Comment by lazymanzy** [comment]
Hi yall! so does any solution to this exist or do you suppose this will get fixed?

**Comment by Thor334433** [comment]
Tried this a couple of days ago and it still appears to be an issue. Everything else is great and this will be the last fix needed in my opinion.

**Comment by eeshaa1** [comment]
SAME ISSUE

**Comment by MechPala** [comment]
Underwater sections are unfortunately unplayable if ultrawide aspect ratio is used:
![image](https://github.com/user-attachments/assets/3d535e49-d67c-4633-8ced-0573151d8858)

**Comment by j0hnnypearl** [comment]
same problem v2.2 mod v.1.5.0 game.

**Comment by j0hnnypearl** [comment]
![20240924203736_1](https://github.com/user-attachments/assets/f50cd4d6-3219-4c87-94ce-368e43234c96)
![20240924202903_1](https://github.com/user-attachments/assets/dcb44217-0b95-43f6-9f7c-536849fe609c)
![20240924203742_1](https://github.com/user-attachments/assets/5de53629-e59d-438c-adb9-af993fbd7cce)

**Comment by SolidSnakeman** [comment]
First off. Thanks for the mod but I have the same problem with version 1.5 of the game and version 2.2 of the mod.

**Comment by devilishwit** [comment]
Glad I found this thread - the mod works damn near perfectly aside from this issue. Managed to screen record an example of the cutscene chicanery courtesy of Vulcan Raven. 

https://github.com/user-attachments/assets/ca0541fd-d506-4a18-b8c8-eb6a2a631872

**Comment by NicoKaneo** [comment]
I don't think the developer gives a shit about fixing it. Or, maybe can't?

**Comment by camirock17** [comment]
> I don't think the developer gives a shit about fixing it. Or, maybe can't?

According to previous post: "Closed as not planned"

**Comment by blanky451** [comment]
I think he wants more Ko-Fi dontations.

**Comment by MechPala** [comment]
> I think he wants more Ko-Fi dontations. 

Would gladly donate if he'd promise to fix this.

**Comment by nuggslet** [comment]
⌛

![Image](https://github.com/user-attachments/assets/51343bbd-2475-4d18-8455-ac47f3741179)

**Comment by blanky451** [comment]
You want to prove me wrong? Do it.

**Comment by blanky451** [comment]
> Want to be mindlessly cynical somewhere else? It's done.

I'm not being mindlessly cynical. The explanation is in the post you quoted. If life gets in the way of projects, I get it. We're all in that boat at some point or another. But you haven't done a release in a year, and you're asking for donations on the project page. That was what led me to say what I did. And I would sincerely like to be proven wrong in this regard.

**Comment by nuggslet** [comment]
What is wrong with people dude? Honestly.

**Comment by nuggslet** [comment]
https://www.reddit.com/r/metalgearsolid/comments/1kpbnh7/mgsm2fix_update_teaser_mgs1_with_fullnative/

**Comment by blanky451** [comment]
> What is wrong with people dude? Honestly.

Seriously? You can't take criticism with valid points?

> https://www.reddit.com/r/metalgearsolid/comments/1kpbnh7/mgsm2fix_update_teaser_mgs1_with_fullnative/

Well. Ok, then. There it is.

**Comment by nuggslet** [comment]
Valid criticism != personal attacks and unfounded assumptions. Bye now.

https://github.com/user-attachments/assets/fa95f4cd-4f89-4344-9b73-64ad01fbe99a

**Comment by blanky451** [comment]
> Valid criticism != personal attacks and unfounded assumptions. Bye now.
> cat-fire.mp4

Later.
